tests/test_onnx_v2.py

```diff
@@ -253,11 +348,44 @@ def test_pytorch_export(self, test_name, name, model_name, feature, onnx_config_
     @require_torch
     def test_pytorch_export_with_past(self, test_name, name, model_name, feature, onnx_config_class_constructor):
         self._pytorch_export(test_name, name, model_name, feature, onnx_config_class_constructor)
+        try:
+            validate_model_outputs(onnx_config, tokenizer, model, output, onnx_outputs, 1e-5)
+        except ValueError as ve:
+            self.fail(f"{name} -> {ve}")
 
+    @slow
+    @require_tf
+    def test_tensorflow_export_with_past(self):
```

Comment:
As above, I wonder if we can wrap this logic into a single function to avoid duplicating the logic

Reply:
Indeed.